pennylane/ops/op_math/condition.py

```diff
@@ -129,13 +141,13 @@ def cond(condition, true_fn, false_fn=None, elifs=()):
     Args:
         condition (Union[.MeasurementValue, bool]): a conditional expression involving a mid-circuit
            measurement value (see :func:`.pennylane.measure`). This can only be of type ``bool`` when
-           decorated by :func:`~.qjit`.
+           decorated by :func:`~.qjit` or when using :func:`~.pennylane.capture.enabled`.

```

Comment:
```suggestion
           decorated by :func:`~.qjit`.
```

Given:

1) capture is an experimental project I don't think we want to publicly expose too much at this point 
2) We will be able to capture `qml.cond(qml.measure(0), qml.RX)(x, 0)` once we can capture MCM's.